src/components/carousel/panels/dataset-info-panel.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react'
+import { DetailsPanel, DetailWrapper, Headline, Description } from './subcomponents/LeftPanel'
+import { DataPanel, DataWrapper, CarouselListItem, CarouselBulletedList } from './subcomponents/RightPanel'
+import { datasetInfoContent } from '../types'
+import { useWindowWidth } from "../../../hooks";
+
+export const DatasetInfoPanel = ({ content }) => {
+  const { isCompact } = useWindowWidth();
+
+  return (
+    <Fragment>
+      <DetailsPanel compact={isCompact}>
```

Comment:
this is a lot of passing `isCompact` around. it's fine, but just verify that all these components actually make use of this prop. i think i found one, but there may be more.